apps/mssql.go

```diff
@@ -18,10 +18,80 @@ import (
 	"context"
 
 	"github.com/GoogleCloudPlatform/ops-agent/confgenerator"
+	"github.com/GoogleCloudPlatform/ops-agent/confgenerator/fluentbit"
 	"github.com/GoogleCloudPlatform/ops-agent/confgenerator/otel"
 	"github.com/GoogleCloudPlatform/ops-agent/internal/platform"
 )
 
+type LoggingProcessorMssqlLog struct {
+	confgenerator.ConfigComponent `yaml:",inline"`
+}
+
+func (LoggingProcessorMssqlLog) Type() string {
+	return "mssql_errorlog"
+}
+
+func (p LoggingProcessorMssqlLog) Components(ctx context.Context, tag string, uid string) []fluentbit.Component {
+	c := confgenerator.LoggingProcessorParseRegex{
```

Comment:
In addition to the Regex, we need the parser shared section that tells fluentbit how to parse the timestamp. the one below is copied from saphana so it's not the right field/format.
```
ParserShared: confgenerator.ParserShared{
			TimeKey:    "time",
			TimeFormat: "%Y-%m-%d %H:%M:%S.%L",
}

Reply:
The original PR calls this out:
```
// Not sending the log timestamp "logtime" from above in as the Time_Key as there will be time zone issues.
// MS SQL Server writes logs in server time, not a standard time zone such as UTC
```
I can add these comments in

Reply:
We'll add the comment and see how the ops-agent team would like to handle it.